components/Card.js

```diff
@@ -0,0 +1,9 @@
+export default function Card({ todo }) {
+	return (
+		<div>
+			<h1>{todo.name}</h1>
```

Comment:
This code renders multiples `<h1>` tags on the page. There should only be a single `<h1>` tag. Please consider using another tag here
```suggestion
			<h2>{todo.name}</h2>
```